lighthouse-core/lib/page-functions.js

```diff
@@ -67,19 +67,30 @@ function registerPerformanceObserverInPage() {
  * @return {Promise<number>}
  */
 function checkTimeSinceLastLongTask() {
-  // Wait for a delta before returning so that we're sure the PerformanceObserver
-  // has had time to register the last longtask
+  // This function attempts to return the time since the last long task occurred.
+  // `PerformanceObserver`s don't always immediately fire though, so we check twice with some time in
+  // between to make sure nothing has happened very recently.
+
+  // Chrome 88 introduced heavy throttling of timers which means our `setTimeout` will be executed
+  // at some point farish (several hundred ms) into the future and the time at which it executes isn't
+  // a reliable indicator of long task existence, instead we check if any information has changed.
```

Comment:
to be fair to chrome, we were relying on something that was never promised, just because you ask for something to be done in 50ms doesn't mean it will happen within 100ms even if everything else is idle, it just *happened* to be true for the past two decades or so :)